Source/Renderer/Texture.js

```diff
@@ -148,6 +148,7 @@ define([
         this._preMultiplyAlpha = preMultiplyAlpha;
         this._flipY = flipY;
         this._sampler = undefined;
+        this.isDefault = false;
```

Comment:
Instead of polluting `Texture` and `Context` with this, can this be handled all in `Material`?